recipes/entt/3.x.x/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "3.12.1":
```

Comment:
This would be a good opportunity to drop some versions. We could leave only the three latest minor versions: 3.12.1, 3.11.1, 3.10.3

Reply:
That is fine by me!